README.md

```diff
@@ -2,22 +2,69 @@
 
 ## Quick start
```

Comment:
Is this section written clearly? I wanted to incorporate both the original `catadog` usage and new Docker usage options.